# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/wxwidgets/libs-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./wx-config.in.orig	2004-11-22 02:40:51.785674568 +0100
+++ ./wx-config.in	2004-11-22 02:41:16.329943272 +0100
@@ -959,7 +959,7 @@
     if [ "x$this_linkage" = "x-static" ]; then
         wx_libs="$wx_libs ${libdir}/libwx_@TOOLCHAIN_NAME@.a $ldlibs_core @LIBS@"
     else
-        wx_libs="$wx_libs -lwx_@TOOLCHAIN_NAME@"
+        wx_libs="$wx_libs -lwx_@TOOLCHAIN_NAME@ $ldlibs_core @LIBS@"
     fi
 
     using_gui=yes
